libs/mock-block-dock/src/service-module-callbacks.ts

```diff
@@ -23,20 +29,39 @@ const callExternalApiMethod = async (params: {
 
   if (!blockProtocolApiKey) {
     throw new Error(
-      `Visit ${bpSiteHost}/dashboard to generate a Block Protocol API key, which is required to make calls to the "${methodName}" method of the BP service module, and ensure it is passed to the \`MockBlockDock\` component as \`${blockProtocolApiKey}\`.`,
+      `Visit ${bpSiteHost}/dashboard to generate a Block Protocol API key, which is required to make calls to the "${methodName}" method of the BP service module, and ensure it is passed to the \`MockBlockDock\` component as blockProtocolApiKey.`,
```

Comment:
Not within the scope of this PR but it'd be really good to have this be visually shown to the user through the DOM rather than buried in the console somewhere like it was last time I tried this out

Reply:
Will this end up called on the Hub if a user doesn't have an API key? Or do we handle that differently because `mock-block-dock` passes control to the app?

Reply:
We will handle that differently because we will implement our own callbacks – this file will be unused